### ISSUE TITLE
Why is there no validation set for pose estimation models ?

### ISSUE THREAD

**Issue by djaym7** [dmlc/gluon-cv#1342]
None of the logs/ training scripts have validation set, just the training scores are given, was wondering the reason behind that..
